study-datastore/src/main/java/com/hphc/mystudies/bean/ReviewBean.java

```diff
@@ -24,18 +24,8 @@
 
 public class ReviewBean {
 
-  private String reasonForConsent = "";
```

Comment:
Please update file copyright to add Google LLC & `2020-2021`

Reply:
Updated copyright.